src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -46,6 +46,18 @@
 register_cli_argument('resource feature list', 'resource_provider_namespace', CliArgumentType(options_list=('--namespace',), required=False, help=_PROVIDER_HELP_TEXT))
 register_cli_argument('resource feature', 'feature_name', CliArgumentType(options_list=('--name', '-n'), help='the feature name'))
 
+register_cli_argument('resource policy', 'resource_group_name', arg_type=resource_group_name_type, completer=get_resource_group_completion_list)
```

Comment:
In that case, you still don't need the completer because it is already part of 'resource_group_name_type'.

---
In reply to: [74480933](https://github.com/Azure/azure-cli/pull/683#discussion_r74480933) [](ancestors = 74480933)